packages/sdk-client-v3/src/middleware/auth-middleware/auth-request-executor.ts

```diff
@@ -64,6 +66,7 @@ export async function executeRequest(options: ExecuteRequestOptions) {
       url,
       method: 'POST',
       headers: {
+        ...request.headers,
```

Comment:
This totally makes sense and explains why the headers were not present on the auth calls! 👏